index.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio Website: Ademola Adebayo</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0/css/all.min.css" integrity="sha512-9usAa10IRO0HhonpyAIVpjrylPvoDwiPUiKdWk5t3PyolY1cOd4DSE0Ga+ri4AuTroPR5aQvXU9xC6qOPnzFeg==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/yourcode.js" crossorigin="anonymous"></script>
+</head>
+<body>
+  <section class="toolbar">
+    <div class="top-toolbar">
+      <p>10:30</p>
+      <div class="top-icons">
+        <i class="	fas fa-signal fa-0.5x"></i>
+        <i class="	fas fa-wifi fa-0.5x"></i>
+        <i class="	fa fa-battery-0 fa-0.5x"></i>       
+      </div>
+    </div>
+    <div class="btm-toolbar">
+      <h2>Smart City</h2>
+      <a href=""><i class=" fas fa-bars fa-2x"></i></a>
+    </div>
+  </section>
+  <section class="headline">
+    <div class="section-head">
+      <h1>Ademola Adebayo</h1>
+      <h3>Software Developer</h3> 
+    <div class="head-text">
+      <p>I can help you build a product,<br> feature or website Look through<br> some of my work and experience! If<br> you like what you see and have 
+        a project you need coded, don’t hestiate to contact me.</p>
+    </div>
+    <hr> 
+    </div>
+    <div class="lang-framework">
+      <div>
+        <p class="lang-col">Language</p>
+        <p>Javascript</p>
+        <p>HTML</p>
+        <p>Ruby</p>
+        <p>CSS</p>  
+      </div>
+      <div>
+        <p class="lang-col">Frameworks</p>
+        <p>Bootstrap</p>
+        <p>Ruby on Rails</p>
+        <p>React</p>
+        <p>Redux</p>
+        <p>State</p>
+      </div>
+    </div>
+    <div class="skills">
+      <p class="lang-col">Skills</p>
+        <p>Codekit</p>
+        <p>Github</p>
+        <p>Codepen</p>
+        <p>Terminal</p>
+        <p>Replit</p>  
+    </div>
+    <hr>
+    <div class="social">
+      <a href="#"><i class="fab fa-github fa-2x"></i></a>
+      <a href="#"><i class="fab fa-facebook fa-2x"></i></a>
+      <a href="#"><i class="fab fa-instagram fa-2x"></i></a>
+      <a href="#"><i class="fab fa-twitter fa-2x"></i></a>
+    </div>
```

Comment:
- [ ] While this is working fine, it's considered good practice to put elements like this inside of an unordered list. Kindly wrap this section inside of a `ul` instead of a `div`, and wrap every child element to be inside of a `li` tag.